lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -435,4 +452,9 @@ def run_commands(module, commands, check_rc=True):
 
 def load_config(module, config, return_error=False, opts=None):
     conn = get_connection(module)
-    return conn.load_config(config, return_error, opts)
+    return conn.load_config(config, return_error=return_error, opts=opts)
```

Comment:
What is the rational for making this change?  `return_error=return_error, opts=opts`.

Reply:
No reason in particular. Reverted that line back to the earlier one.